### PR TITLE
Add awaitMessage function

### PR DESCRIPTION
**Pull request by u9g**

### CODE REVIEW

**Comment by BlueBurgersTDD** [comment]
https://github.com/u9g/mineflayer/pull/1 possibly add regex support?

**Comment by u9g** [comment]
now supports regex thanks to @BlueBurgersTDD :)

**Review by DrMoraschi — APPROVED** [review]

**Review by BlueBurgersTDD — APPROVED** [review]

File: test/externalTests/chat.js [hunk 1]
```diff
@@ -60,5 +60,15 @@ module.exports = () => {
...
+  addTest('test awaitMessage', async (bot) => {
+    // let resolves = 0
+    const p = bot.awaitMessage('<flatbot> hello')
+    bot.chat('hello')
+    await p
+    const p2 = bot.awaitMessage(['<flatbot> hello', '<flatbot> world'])
+    bot.chat('world')
+    await p2
+  })
+
```
Comment (BlueBurgersTDD) [inline test/externalTests/chat.js:72]: maybe add a test for regex?

**Review by aesthetic0001 — APPROVED** [review]

**Comment by u9g** [comment]
ready for review :)

**Review by rom1504 — APPROVED** [review]

**Pull request merged**